src/Host/Views/Consent/Index.cshtml

```diff
@@ -7,10 +7,12 @@
             {
                 <div class="client-logo"><img src="@Model.ClientLogoUrl"></div>
             }
-            <h1>
-                @Model.ClientName
-                <small>is requesting your permission</small>
-            </h1>
+            <h1>@Html.Raw((string.IsNullOrWhiteSpace(Model.ClientUrl))? $"{@Model.ClientName}" : $"<a href='{@Model.ClientUrl}'>{@Model.ClientName}</a>")                </h1>
```

Comment:
I don't like Html.Raw in here -- XSS potential.

Reply:
I haven't done web development in 10 years so i am a bit behind in what one should be using.

Do you have a suggestion?

Reply:
I'd just write the if/else blocks to build the html.

Reply:
I wont have time to do this until Thursday.  Up to you if you can wait it looks like your on a clean out PR spree today.

Reply:
No, I am just looking at issues. No time yet for me to do any code :)